dff/pipeline/service/extra.py

```diff
@@ -147,8 +151,9 @@ async def __call__(self, ctx: Context, actor: Actor, component_info: ServiceRunt
     def info_dict(self) -> dict:
         """
         Property for retrieving info dictionary about this wrapper.
-        Returns info dict, containing its fields as well as its type.
-        All not set fields there are replaced with '[None]'.
+
+        :return: Info dict, containing its fields as well as its type.
+            All not set fields there are replaced with `[None]`.
```

Comment:
Shouldn't it be just
```suggestion
            All not set fields there are replaced with `None`.
```